src/structures/vroom/input/input.cpp

```diff
@@ -678,7 +678,7 @@ void Input::set_matrices(unsigned nb_thread) {
           // Durations matrix not manually set so defined as empty
           // above.
           if (_locations.size() == 1) {
-            d_m->second = Matrix<Cost>({{0}});
+            d_m->second = Matrix<Cost>(0);
```

Comment:
We should create a one-element matrix with a zero here, not an empty matrix, see https://github.com/VROOM-Project/vroom/issues/699#issuecomment-1106822394

Reply:
Alright. I've updated the code.